org.eclipse.ice.data.neutrons/frontend/styles/org/eclipse/ice/README

```diff
@@ -0,0 +1,3 @@
+Place your optional styles in this folder.
```

Comment:
Is there better text for this README too?

It is OK to have the default text, but we should consider adding some context for why these are here if we are going to keep them. Something like "This file is auto-generated by our project generators. It is for Vaadin. Blah blah blah..."